src/embedded-checkout/iframe-content/iframe-embedded-checkout-messenger.ts

```diff
@@ -113,7 +113,7 @@ export default class IframeEmbeddedCheckoutMessenger implements EmbeddedCheckout
                 const handler = this._messageHandlers[key as keyof EmbeddedCheckoutEventMap];
 
                 if (handler) {
-                    handler.call(null, message);
+                    (handler as any).call(null, message);
```

Comment:
I tried to avoid using any in all these cases but the solution didn't seem straight forward. Looked like a types refactor would be required to avoid `any`.

Reply:
Hrmm yeah I'm not sure what's going on. Anyway, it might slightly better if you write `as (event: EmbeddedCheckoutEvent) => void` instead.